static/app/components/badge/tag.tsx

```diff
@@ -156,13 +150,16 @@ export default Tag;
 
 const TAG_HEIGHT = '20px';
 
-export const Background = styled('div')<{type: keyof Theme['tag']}>`
+export const Background = styled('div')<{
+  type: keyof Theme['tag'];
+  borderStyle?: TagProps['borderStyle'];
```

Comment:
Made this optional since other components were complaining that this was required, not sure if this has any unintentional consequences though.